src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +86,101 @@ Common questions and requests include:
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
+
+Feedback is a gift. Every piece of meaningful feedback represents someone taking time out of their day to express something to us. It's important that we honor that gift. The sidekick role was created so that we have a role and a process for getting a human being to read through and think about and sometimes respond to every piece of feedback that people write to us. 
+
+Customer feedback from our docs site creates tickets in Jira. It's the sidekick's responsibility to look at every ticket that comes in (usually about 5–10 per day) and make decisions about each one.
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Many pieces of customer feedback simply aren't actionable. That's ok! Even if we can't act on a specific piece of feedback, we can use this feedback in aggregate to get an overall sense of how our docs site or a specific category of our docs site is landing with our readers.
+
+Here are some fictional examples of feedback we might not be able to take action on specifically:
+
+`These docs are bad.`
+
+or 
+
+`Help!`
+
+There's also feedback that falls into the category of nonsense or mistakes.
+
+Follow this process to close tickets that aren't actionable or don't make sense:
+
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+2. Set the Workflow status to **Closed** 
+3. Set the Resolution to **Cannot Reproduce**, **Filed by Mistake**, or **Won't Do** depending on what feels most appropriate.
+4. Add a brief note about why you closed it. Usually, "Not actionable" is plenty.
+
+### Fix customer feedback
+
+Sometimes, though rarely, a customer feedback ticket comes in that's highly actionable and is a quick fix, such as a broken link or an outdated image. 
+
+Follow this process to close these:
+
+1. Create a PR to address the issue.
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
```

Comment:
Is there a way we can describe this? I see it as taking the first "clause" (?) of a URl that follows the `/docs/` part of the URL. Something like "Add a category label based on the URL, which will generally be whatever category follows the /docs/ part of the URL" (but better and more cogent maybe.)

Reply:
Good call!